fairlearn/metrics/extra_metrics.py

```diff
@@ -1,9 +1,29 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+"""This module contains a variety of extra metrics which are useful for assessing fairness
```

Comment:
I'm open to other names beyond the rather generic `extra_metrics`

Reply:
extra? As in additional? Or should we just have a separate file for each of these.

Reply:
Well, the naming question is no longer relevant....